### ISSUE TITLE
SSH._proxyCommand not initialized properly

### ISSUE THREAD

**Issue by baileytj3** [alexforster/sisqo#1]
In SSH.py the line 

```
  self._proxyCommand = str(proxyCommand)
```

 will result in the string 'None' being stored in the _proxyCommand variable when the default parameter is used. This causes this line

```
   self._proxySocket = paramiko.ProxyCommand(proxyCommand) if proxyCommand else None
```

 in the __ enter __ method to fail with the error 'OSError: [Errno 2] No such file or directory'. Here proxyCommand is not None, but actually contains the string 'None'.

**Comment by alexforster** [comment]
Fixed and uploaded to PyPI as v1.0.3.

Edit: thanks, by the way! :)

**Comment by baileytj3** [comment]
No problem. Appreciate the module!
